app/components/header.tsx

```diff
@@ -1,11 +1,78 @@
-import { Text, Center } from '@chakra-ui/react';
+import {
+  Text,
+  Heading,
+  Avatar,
+  Flex,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+} from '@chakra-ui/react';
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { Link, useFetcher } from '@remix-run/react';
+
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
+  const fetcher = useFetcher();
   return (
     <header>
-      <Center>
-        <Text fontSize="2xl">Hi! I am the header</Text>
-      </Center>
+      <Flex
+        backgroundColor="brand.500"
+        height="70px"
+        alignItems="center"
+        shadow="2xl"
+        border="2px"
+        fontSize="18"
+      >
+        <HStack width="50%" color="white" justifyContent="flex-start" gap="10" marginLeft="10">
+          <Link to={{ pathname: '/domains' }}>
+            <Flex alignItems="center">
+              <TriangleUpIcon marginRight="2" />
+              <Text>Custom Domain</Text>
```

Comment:
Should this be `Custom Domains` instead of `Custom Domain`?

Reply:
Or better: `Domains`.  All domains are custom, so it's redundant and takes more space.

Reply:
Yea, that is much better

Reply:
![image](https://user-images.githubusercontent.com/32577022/218167107-4e4f6a85-00e6-434b-84ad-e38a60ae6f3a.png)
Should I indent it more or should I keep it as is?

Reply:
That looks good to me